mastodon.js

```diff
@@ -21,55 +21,57 @@ function get (path,params) {
 		log.info("get",path,params)
 }
 
+function genShortURL(){
+	// Were assuming single thread sequential URL generation here...
+	let num = Math.floor(Date.now()/1000)-1669485542;
+	let digits = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+	let len = Math.min(digits.length, 62);
+	let result = '';
+	while (num > 0) {
+		result = digits[num % len] + result;
+		num = parseInt(num / len, 10);
+	}
+	return result;
+}
+
```

Comment:
Perhaps this should just use an existing base64 implementation, rather than rolling our own base62 one?

Is `parseInt(x / y, 10)` a standard JS idiom for integer division?  It seems like it's going to mean an double to string then a string to int conversion, though maybe the implementation is smart enough to spot and optimise this case.

Reply:
I want base62 so I can use it as filenames